hedera-node/configuration/mainnet/bootstrap.properties

```diff
@@ -1,2 +1,17 @@
 # This file needs to exist even though there aren't any properties in it. Do NOT put properties in this file.
 # Downstream processes depend on its existence.
+# This is an exact replica of file 121 on mainnet and must match application.properties exactly.
+# It will be used by mono-service for all the property overrides.
+# This file will be deprecated when we deprecated using mono-service config and fully move to hedera-config.
+autoRenew.targetTypes=
+# Of historical interest only---block 33476932 was the "synchronization block" used to align the consensus
+# node block numbers with mirror node blocks in the 0.26 Services release
+contracts.knownBlockHash=055fac8eb788872957dfee45b7d40f7c94d66ea0a75ec703b993d57e441de574@33476932
+contracts.localCall.estRetBytes=4096
+hedera.allowances.isEnabled=true
+scheduling.whitelist=CryptoTransfer,ConsensusSubmitMessage,TokenBurn,TokenMint,CryptoApproveAllowance
+staking.fees.stakingRewardPercentage=10
+staking.maxStakeRewarded=650000000000000000
+staking.rewardBalanceThreshold=8500000000000000
+tokens.nfts.maxAllowedMints=10_000_000
+accounts.maxNumber=20_000_000
```

Comment:
This is all that is in file 121 on mainnet today?

Reply:
Yes, pulled file using yahcli and copy pasted